playbooks/gluster-cluster/cluster-vars.yml

```diff
@@ -47,7 +47,6 @@ gluster_infra_fw_ports:
   - 2049/tcp
   - 54321/tcp
   - 5900/tcp
-  - 5900-6923/tcp
```

Comment:
Does this mean the port from 5900-6923 are not needed, because in line49 5900 still exist @parthdhanjal

Reply:
So we just don't need 6900/tcp port
Because that is required by Hosted Engine